gravityflow/gravityflow
# Feature/status entry url filter

[#HS 14297](https://secure.helpscout.net/conversation/1333659385/15380/#)

## Description
The status view lacked a filter to customize what URL the user would be sent to when displayed via the shortcode. Similar to gravityflow_entry_url_inbox_table for the inbox view.

## Testing instructions
- Create a page with a [gravityflow page="status"] shortcode on it.
- Add the following snippet to your functions.php file and adjust the page / get_the_ID() control for your environment.
```
add_action( 'gravityflow_entry_url_status_table', 'jo_status_entry_url_blank', 10, 4 );
function jo_status_entry_url_blank( $entry_url, $form_id, $entry_id, $entry ) {
	if ( ! get_the_ID() == '285' ) {
		return $entry_url;
	} else {
		return '#';
	}  
}
```
- Verify that the link can be clicked but user does not navigate away from the status list view.
 
## Documentation Changes?
Yes - Draft entry for gravityflow_entry_url_status_table has been prepared in Developers / Actions & Filters category.

## Checklist:
